config/env/default.js

```diff
@@ -28,5 +28,19 @@ module.exports = {
   sessionKey: 'sessionId',
   sessionCollection: 'sessions',
   logo: 'modules/core/client/img/brand/logo.png',
-  favicon: 'modules/core/client/img/brand/favicon.ico'
+  favicon: 'modules/core/client/img/brand/favicon.ico',
+  uploads: {
+    profileUpload: {
+      dest: './modules/users/client/img/profile/uploads/', //Profile upload destination path
+      limits: {
+        fileSize: 1*1024*1024 //Max file size in bytes
+      },
+      fileFilter: function (req, file, cb) {
+        if (file.mimetype !== 'image/png' && file.mimetype !== 'image/jpg' && file.mimetype !== 'image/jpeg' && file.mimetype !== 'image/gif') {
+            return cb(new Error('Only image files are allowed!'), false);
+        }
+        cb(null, true);
+      }
+    }
```

Comment:
First of all, this is indeed a server-side config thing.
Secondly, there's another down-side to put it like that which is that we now force that filefilter everywhere else multer is used and that is wrong because for uploading some other kind of assets you wouldn't want to force only images.

@gustavodemari please recommend another alternative.